src/ecobalyse_data/food/export_agb/README.md

```diff
@@ -0,0 +1,104 @@
+# Exporter la base agribalyse de brightway vers des fichiers json
+
+Au préalable, vérifiez que vous avez bien suivi les instructions du
+[README](../README.md) général, et que vous avez installé le projet comme
+indiqué dans le [README](../../../../README.md) du repository.
+
+## Exporter les données pour l'explorateur de produits CIQUAL
+
+Lancer le script d'export qui peut prendre plusieurs heures (!) :
+
+    $ python export_ciqual.py
+
+Il est possible d'utiliser l'option `--max` pour limiter le nombre de produits
+ciquals à exporter, et l'option `--no-impacts` pour ne pas calculer les impacts
+des procédés (ce qui exportera un fichier `processes-no-impacts.json` au lieu de
+`processes.json`) :
+
+    $ python export_ciqual.py --max 1 --no-impacts
+    # Beaucoup plus rapide, mais incomplet ;)
+
+Les fichiers résultants sont `/public/data/food/processes/explorer.json` et
+`/public/data/food/products.json`.
+
+Optionnellement, lancer le script de vérification des différences d'impacts :
+
+    $ python checks_ciqual.py
+
+## Exporter les données pour le constructeur de recettes
+
+Lancer le script d'export :
+
+    $ python export_builder.py
+
+Exemple :
+
+    $ python export_builder.py
```

Comment:
These two command lines are the same.

Also, when running this I'm getting `AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')`:

```
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) python export_ciqual.py --max 1 --no-impacts
Get ciqual codes from ../Agribalyse_Synthese.csv
Open the agribalyse3 brightway database
Biosphere database already present!!! No setup is needed
Search for the ciqual products in the brightway database
Loaded 1 products
Building product tree
100%
Export de 1 produits vers ../../../../public/data/food/products.json
Export de 17 produits vers processes-no-impacts.json
Terminé.
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) ✗ python checks_ciqual.py
>>> Liste des produits avec étapes manquantes
0 missing steps
(ecobalyse-env) ➜  export_agb git:(merge-ecobalyse-data) ✗ python export_builder.py
10 procédés à rajouter provenant de ingredients_base.json (ingrédients complexes)
Total de 54 procédés à exporter
Biosphere database already present!!! No setup is needed
Chargement de 54 activités terminé
Total de 54 activités trouvées dans agribalyse
Initialisation de la méthode ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
Traceback (most recent call last):
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/export_agb/export_builder.py", line 342, in <module>
    lcas = init_lcas({random_process: 1})
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/export_agb/export_builder.py", line 151, in init_lcas
    lca = bw.LCA(demand, method)
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/lca.py", line 106, in __init__
    self.get_array_filepaths()
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/lca.py", line 124, in get_array_filepaths
    get_filepaths(self.method, "method"),
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/site-packages/bw2calc/utils.py", line 98, in get_filepaths
    assert name in metadata, "Can't find {} object {}".format(kind, name)
AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
```

Reply:
Interesting comment from @ccomb over Mattermost regarding this error : 

> Et au passage j'ai la confirmation qu'avec bw2io=0.8.6 (ou 7) ça passe, et avec 0.8.8 ça refait l'erreur AssertionError: Can't find method object ('EF v3.0', 'acidification', 'accumulated exceedance (ae)')
> Donc pour l'instant je bloque en 0.8.7

We may want to ensure dependency versions are pinned.